src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier randomColor = new ColorSupplier();
+    private final Random random = new Random();
+    private final int numbersLimit = 100;

```

Comment:
see the right constant naming: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names